targets/unity-v2/Testing/Tests/Client/LightweightEventsTests.cs

```diff
@@ -1,4 +1,6 @@
-#if NET_4_6
+#if !NET_4_6 && (NET_2_0_SUBSET || NET_2_0)
```

Comment:
Did we do regression test on Unity 2018 and make sure it's not broken ?

Reply:
Everything is working.